vb_suite/io_bench.py

```diff
@@ -1,8 +1,11 @@
 from vbench.api import Benchmark
 from datetime import datetime
 
-common_setup = """from pandas_vb_common import *
-from StringIO import StringIO
+common_setup = """from .pandas_vb_common import *
+try:
+    from StringIO import StringIO
+except ImportError:
```

Comment:
see #10470 -- you can probably just do `from io import StringIO` without the try-except.